homeassistant/components/life360/device_tracker.py

```diff
@@ -266,8 +270,20 @@ def _update_member(self, member, dev_id):
             ATTR_WIFI_ON: _bool_attr_from_int(loc.get('wifiState')),
         }
 
-        self._see(dev_id=dev_id, gps=(lat, lon), gps_accuracy=gps_accuracy,
-                  battery=battery, attributes=attrs,
+        # If user wants driving or moving to be shown as state, and current
+        # location is not in a HA zone, then set location name accordingly.
+        loc_name = None
+        active_zone = run_callback_threadsafe(
```

Comment:
For a future PR, this should all become async.

Reply:
Yep